### PR TITLE
Feature/update error icons and placement in form components

### PR DESCRIPTION
**Pull request by eskilsen**
Update error icons and error messages to be more wcag compliant. 
Users with color disabilities would have struggled or better yet, not noticed the old errors. 
This is now fixed. 

## How Has This Been Tested?

Updated tests for some components and updated snapshots. 
## Screenshots (if appropriate):
image

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [X] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

- [X] My code follows the code style of this project.
- [X] My change requires a change to the documentation.
- [X] I have updated the documentation accordingly.
- [ ] I have read the **CONTRIBUTING** document.
- [X] I have updated the **CHANGELOG** document.
- [X] I have added tests to cover my changes.
- [X] All new and existing tests passed.

## Review instructions
[Review instructions](../REVIEW_INSTRUCTIONS.md)

### CODE REVIEW

File: src/App/components/FormComponents/Checkbox.js [hunk 1]
```diff
@@ -26,7 +26,7 @@ const Checkbox = ({ id, checked, name, disabled, groupTitle, label, errorMessage
                         <input {...attrs} id={id} />{"\n"}
                         <label htmlFor={id}>{label}</label>{"\n"}
                     </div>)}
-                    {errorMessage && <><div className="help-block">{errorMessage}</div>{"\n"}</>}
+                    {errorMessage && <><div className="help-block"><i className="material-icons">error</i>{errorMessage}</div>{"\n"}</>}
```
Comment (PhungNg) [inline src/App/components/FormComponents/Checkbox.js:29]: According to figma, the icon should be a triangle 
image

File: src/App/components/FormComponents/Radio.js [hunk 2]
```diff
@@ -24,7 +24,7 @@ const Radio = ({ id, checked, disabled, label, group, groupTitle, options, optio
                             <input {...attrs} id={id} defaultChecked={checked}/>{"\n"}
                             <label htmlFor={id}>{label}</label>{"\n"}
                         </div>)}
-                        {errorMessage && <><div className="help-block">{errorMessage}</div>{"\n"}</>}
+                        {errorMessage && <><div className="help-block"><i className="material-icons">error</i>{errorMessage}</div>{"\n"}</>}
```
Comment (PhungNg) [inline src/App/components/FormComponents/Radio.js:27]: Triangle icon in figma
image

File: src/App/components/InputGroup/index.js [hunk 3]
```diff
@@ -118,7 +117,7 @@ const InputGroup = ({
                             <input {...attrs} />}{"\n"}
                 </>
             }
-            {errorMessage && <><div className="help-block">{errorMessage}</div>{"\n"}</>}
+            {errorMessage && <><div className="help-block"><i className="material-icons">error</i>{errorMessage}</div>{"\n"}</>}
```
Comment (PhungNg) [inline src/App/components/InputGroup/index.js:120]: The icon should be inside the field, and not before the error message. Like this:
image

This also removed the error icon from input field
image

File: src/less/components/validation.less [hunk 4]
```diff
@@ -63,16 +63,6 @@
...
 
-        input {
-            &:not(:focus) {
-                padding-right: 2.5rem;
-                background-image: url("../../icons/shapes/warning.svg");
-                background-size: 24px 24px;
-                background-repeat: no-repeat;
-                background-position: calc(100% - 0.5rem) 50%;
-            }
-        }
```
Comment (PhungNg) [inline src/less/components/validation.less:74]: Why are this removed?

**Review by PhungNg — CHANGES_REQUESTED** [review]
A few things to change. Also, the Select component also need an error icon inside the input

**Comment by eskilsen** [comment]
I think there are some misunderstandings regarding which figma file to follow 😅 
I followed this https://www.figma.com/file/1OlqVHKlbnyl63sYkaL2WI/%C3%85.-Pattern%2FForms?node-id=122%3A3414 which was written in the jira-task https://payexjira.atlassian.net/jira/software/c/projects/SWED/boards/312?modal=detail&selectedIssue=SWED-1732

**Comment by PhungNg** [comment]
Ok, good thing we got it straight at least :)
Not much to say other then good job then! :D

**Review by PhungNg — APPROVED** [review]
Woho! :D

**Pull request merged**